packages/expo-web-browser/src/WebBrowser.types.ts

```diff
@@ -8,7 +8,7 @@ export type WebBrowserWindowFeatures = Record<string, number | boolean | string>
 // @needsAudit
 export type WebBrowserOpenOptions = {
   /**
-   * Color of the toolbar in either `#AARRGGBB` or `#RRGGBB` format.
+   * Color of the toolbar.
```

Comment:
```suggestion
   * Color of the toolbar in either `#AARRGGBB` or `#RRGGBB` format or the color name from the [CSS3 specification](https://www.w3.org/TR/css-color-3/#svg-color).
```

Reply:
It does also supports things like `rgb(0,0,0)`, maybe it would be better to link to RN colors docs (https://reactnative.dev/docs/colors)?